examples/misc/simple-map.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Simple map | CARTO</title>
```

Comment:
I think the title is not very descriptive.

maybe `no basemap` ? 🤔

Reply:
I'd say `standalone`

Reply:
This is a custom carto.Map used now for the tests. May be we shouldn't include it now in the examples.